src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -40,6 +40,8 @@
 import pytest
 from psyclone.transformations import OCLTrans
 from psyclone_test_utils import get_invoke
+from psyclone.gocean1p0 import GOKernelSchedule
```

Comment:
pylint complains that the imports from `psyclone` are not grouped.

Reply:
Grouped together `psyclone` imports.